build-system/dep-check-config.js

```diff
@@ -251,4 +251,12 @@ exports.rules = [
       'src/iframe-helper.js',
     ],
   },
+
+  {
+    filesMatching: 'extensions/amp-ad-network-*/**/*.js',
```

Comment:
Any reason we can't just make this filesMatching: '**/*.js'?

Reply:
sure we can do that.